ext/resourcemgr/optimus_test.go

```diff
@@ -35,9 +35,9 @@ func (o *OptimusResourceManager) TestGetJobSpecifications() {
 		}
 
 		var ctx context.Context
-		var filter models.JobSpecFilter
+		var unresolvedDependency models.UnresolvedJobDependency
```

Comment:
would be good if rename could be a separate commit

Reply:
got it, will try to separate commits like this in the future. but for this one i am resolving as we are not only renaming.